### DISCUSSION TITLE
LOTW upload from Cloudlog - how does it work?

### DISCUSSION THREAD

**Discussion by 205er** [magicbug/Cloudlog#2065] [Q&A] [answered]
Hi all,

I setup my own Cloudlog instance just a couple of days ago. Everything is working fine and I am really satisfied with this log book solution :-)
Unfortunately, I seem to be a bit to stupid to get the LOTW interface working. After I applied for a LOTW account and got my certificate, I immediately tried to configure the Cloudlog integration. This does only work in one direction: At the moment I am able to download data from my LOTW account which leads to QSO confirmations in my log book. But the upload for logged QSOs is not working. Or I am not smart enough to figure out how to do that? Or is there really only the manual way by exporting the ADIF file?
Is there anyone out who can give me some guidance?  Checking several log files on my webserver did not reveal any obvious defects in my Cloudog setup. ..and reading the documentation did not take me any further... 

Thanks in advance for your feedback.

**Comment by phl0** [comment]
Did you upload your LotW certificate (properly)?

**Reply by 205er** [reply]
Yes I did as the following screenshot shows:
![image](https://user-images.githubusercontent.com/14996169/232807624-be41fdf3-d4dc-458a-afb0-1cbdb2c91fda.png)

**Reply by phl0** [reply, marked as answer]
You set the wrong DXCC for your certificate. DL (DXCC 230) is FEDERAL REPUBLIC OF GERMANY.

**Comment by 205er** [comment]
You are right! The DXCC location needs to be set during cert upload to Cloudlog. Seems that I picked the wrong entry here...
Reupload of the cert with corrected DXCC worked for me. Upload of QSO is now possible,
